### PR TITLE
modified migration section to add new migration tab feature

### PR DESCRIPTION
**Pull request by emuus**
-added new section "automated migration" for new migration feature
-differentiate scan/migration
-new screenshots

### CODE REVIEW

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+.. _migration-automatic:
+
+Migrating Using Automatic Process
+---------------------------------
+
+You can use the ``Migrations`` tab to complete a migration automatically. When you run an automatic migration, you cannot modify any of the  (this is also referred to as "black box migration").
+
+.. note:: The ``Migrations`` tab for automated migration is only supported for the following target platforms: K5, Microsoft Azure, OpenStack, SUSE Cloud, VMware Vcenter.
+
+.. note:: Migration using the ``Migrations`` page is not currently possible. Refer to :ref:`migration-prereq`.
```
[the remark this answers is not in the document]
Comment (emuus) [reply end-user/en/source/pages/migration/automatic-migration.rst]: Should be "White box migration using "Migrationgs" tab is not supported"

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.. Copyright 2018 FUJITSU LIMITED
+
+.. _migration-automatic:
+
+Migrating Using Automatic Process
+---------------------------------
+
+You can use the ``Migrations`` tab to complete a migration automatically. When you run an automatic migration, you cannot modify any of the  (this is also referred to as "black box migration").
```
[the remark this answers is not in the document]
Comment (emuus) [reply end-user/en/source/pages/migration/automatic-migration.rst]: oups !

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+.. note:: The ``Migrations`` tab for automated migration is only supported for the following target platforms: K5, Microsoft Azure, OpenStack, SUSE Cloud, VMware Vcenter.
```
[the remark this answers is not in the document]
Comment (emuus) [reply end-user/en/source/pages/migration/automatic-migration.rst]: 3 might be too many. I've removed 1 but kept the 2 others as someone might skip over the intro section, straight to the "procedure" section

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+.. note:: Migration using the ``Migrations`` page is not currently possible. Refer to :ref:`migration-prereq`. 
+
+This process includes the following steps:
```
[the remark this answers is not in the document]
Comment (emuus) [reply end-user/en/source/pages/migration/automatic-migration.rst]: yes of course

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+	2. Click on the ``migrate`` button on the top right.
+
+	.. image:: /images/migration-launch.png
+
+	3. Enter the name of your migration. 
+	4. From the drop-down menu select the target platform and type. Enter all other required fields, based on the target you have selected.
+
+	.. note:: The ``Migrations`` tab for automated migration is only supported for the following target platforms: K5, Microsoft Azure, OpenStack, SUSE Cloud, VMware Vcenter.
+
+	5. Click ``launch``.
```
[the remark this answers is not in the document]
Comment (emuus) [reply end-user/en/source/pages/migration/automatic-migration.rst]: ok

File: end-user/en/source/pages/migration/overview.rst [hunk 6]
```diff
@@ -5,9 +5,14 @@
 Migrating Live Workloads
 ========================
 
-UForge AppCenter offers the capability to migrate a live system. UForge Migration will "deep scan" a live system and report back the meta-data of every file and package that makes up the running workload. It also allows you to change or add individual components prior to the final migration.
+UForge AppCenter offers the capability to migrate a live system using two different methods:
 
-The following sub-sections describe in detail the scan steps and the differences between black box and white box migration:
+* using ``Scans``. In this case there are three steps to migration: scan, generate and publish. Each step is done seperately by the user. When using scan to migrate, you can modify the appliance prior to the final migration.
+* using ``Migration``. This automated process is only possible for certain formats.
```
[the remark this answers is not in the document]
Comment (emuus) [reply end-user/en/source/pages/migration/overview.rst]: ok

**Comment by jfknoepfli** [comment]
Duplicated by #639. Closing this one.

**Pull request closed without merging**